### PR TITLE
Update installation instructions

### PR DESCRIPTION
**Pull request by mikemhenry**
Fixes issue #759﻿

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -43,7 +22,7 @@ You can install the latest stable release build of perses via the ``conda`` pack
 
 .. code-block:: none
 
-   $ conda config --add channels omnia
+   $ conda config --add channels conda-forge
```
Comment (jchodera) [inline docs/installation.rst:25]: We also need to mention the openeye-toolkits installation step if we don't already!
```bash
conda install -c openeye openeye-toolkits
```
Or, if we want folks to be able to `conda update` everything, including the toolkit, we should add `openeye` to the `conda config --add channels` and then add `conda install --yes openeye-toolkits` somewhere.

**Review by jchodera — CHANGES_REQUESTED** [review]
Thanks for updating this!

File: docs/installation.rst (hunk 1 above)
Comment (mikemhenry) [reply docs/installation.rst]: good catch! I added `openeye` to the list of channels to add

File: docs/installation.rst [hunk 2]
```diff
@@ -43,7 +22,7 @@ You can install the latest stable release build of perses via the ``conda`` pack
 
 .. code-block:: none
 
-   $ conda config --add channels omnia
+   $ conda config --add channels conda-forge openeye
```
Comment (jchodera) [inline docs/installation.rst:25]: I think we need to still explicitly advise the user install `openeye-toolkits` if we don't currently do so.

Comment (mikemhenry) [reply docs/installation.rst:25]: You are correct!

**Pull request merged**